Dockerfile

```diff
@@ -126,6 +126,19 @@ RUN chmod +x /usr/local/bin/docker-entrypoint.sh
 RUN ln -s /usr/bin/timescaledb-tune /usr/local/bin/timescaledb-tune
 RUN ln -s /usr/local/bin/docker-entrypoint.sh /docker-entrypoint.sh
 
+# hot-forge is a project that allows hot-patching of postgres containers
+# It is currently a private timescale project and is therefore not included/built by default,
+# and never included in the OSS image.
+ARG TIMESCALE_HOT_FORGE=
+RUN if [ ! -z "${PRIVATE_REPO_TOKEN}" -a -z "${OSS_ONLY}" -a ! -z "${TIMESCALE_HOT_FORGE}" ]; then \
+        GH_REPO="https://api.github.com/repos/timescale/hot-forge/"; \
+        ASSET_ID="$(curl -sL --header "Authorization: token ${PRIVATE_REPO_TOKEN}" "${GH_REPO}/releases/tags/${TIMESCALE_HOT_FORGE}" | jq '.assets[0].id')"; \
+        curl -sL --header "Authorization: token ${PRIVATE_REPO_TOKEN}" \
+                 --header 'Accept: application/octet-stream' \
+                 "${GH_REPO}/releases/assets/40428091" > /usr/local/bin/hot-forge || exit 1; \
```

Comment:
We seem to forget a hard-coded asset ID here @feikesteenbergen ?

Reply:
Ah, sorry about that. Will fix